pkg/overlay/cache.go

```diff
@@ -130,3 +131,16 @@ func (cache *Cache) Put(ctx context.Context, nodeID storj.NodeID, value pb.Node)
 
 	return cache.db.Put(nodeID.Bytes(), data)
 }
+
+// ConnFailure implements the Transport Observer `ConnFailure` function
+func (cache *Cache) ConnFailure(ctx context.Context, node *pb.Node, err error) {
+	// noop until we figure out what to do on ConnFailure
```

Comment:
maybe add TODO keyword in case we grep on that again in the future

Reply:
Could you put the following code in here?

```
_, err := cache.statdb.UpdateUptime(ctx, node.Id, false)
if err != nil {
	// log err
}
```

Reply:
You got it 👍